docs/azure/container-registry-integration.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Azure Container Registry integration
+description: Learn how to integrate Azure Container Registry with .NET Aspire for secure container image management.
+ms.date: 05/09/2025
+---
+
+# .NET Aspire Azure Container Registry integration
```

Comment:
Should we note somewhere here that it is in preview?

Reply:
I missed your comment about it being preview in the original issue, sorry. Updated.